pkg/subctl/cmd/validate.go

```diff
@@ -21,9 +21,9 @@ import (
 
 var (
 	validateCmd = &cobra.Command{
```

Comment:
Don't we have to change this also? and package name too? Or am I missing something?

Reply:
yeah we talked about this on slack - all of that is internal - I'll push a PR to rename the files, vars etc - let's get the external changes in for 0.9.0.